src/miniscript/musig_key.rs

```diff
@@ -0,0 +1,102 @@
+//! Support for multi-signature keys
+use std;
+use std::str::FromStr;
+use std::{error, fmt};
+
+use crate::expression::Tree;
+use crate::MiniscriptKey;
+
+#[derive(Debug, Clone, PartialEq, PartialOrd, Ord, Eq, Hash)]
+/// Enum for representing keys in miniscript
+pub enum KeyExpr<Pk: MiniscriptKey> {
```

Comment:
Can you manually implement Debug?

Reply:
Shouldn't the output of debug be same as display? If not, what should be the format while debugging ?
For example if the key is `musig(A,musig(B,C),musig(D))` what should debug print for this ?